pkg/apis/nfd/v1alpha1/expression.go

```diff
@@ -30,17 +30,17 @@ import (
 )
 
 var matchOps = map[MatchOp]struct{}{
-	MatchAny:          struct{}{},
-	MatchIn:           struct{}{},
-	MatchNotIn:        struct{}{},
-	MatchInRegexp:     struct{}{},
-	MatchExists:       struct{}{},
-	MatchDoesNotExist: struct{}{},
-	MatchGt:           struct{}{},
-	MatchLt:           struct{}{},
-	MatchGtLt:         struct{}{},
-	MatchIsTrue:       struct{}{},
-	MatchIsFalse:      struct{}{},
+	MatchAny:          {},
+	MatchIn:           {},
+	MatchNotIn:        {},
+	MatchInRegexp:     {},
+	MatchExists:       {},
+	MatchDoesNotExist: {},
+	MatchGt:           {},
+	MatchLt:           {},
+	MatchGtLt:         {},
+	MatchIsTrue:       {},
+	MatchIsFalse:      {},
```

Comment:
nit: again, all changes in this file make sense but are unrelated (and could be just submitted as a separate PR for speeding up reviews)

Reply:
yeah I agree with you. Will decouple those changes.